lib/rmt/config.rb

```diff
@@ -3,15 +3,34 @@
 
 Config.setup do |config|
   config.merge_nil_values = false
+
+  unless ENV['RAILS_ENV'] == 'test'
+    config.schema do
+      required(:scc).schema do
+        required(:username).filled
+        required(:password).filled
+      end
+    end
+  end
 end
 
 Config.load_and_set_settings(
+  File.join(__dir__, '../../config/default_rmt.yml'),
   '/etc/rmt.conf',
   File.join(__dir__, '../../config/rmt.yml'),
   File.join(__dir__, '../../config/rmt.local.yml')
 )
 
 module RMT::Config
+  def self.method_missing(key)
+    raise NoMethodError if Settings[key].nil?
+    Settings[key]
+  end
+
+  def self.respond_to_missing?(method, *args)
+    method == :scc or super
```

Comment:
Seems like this shouldn't be needed in the current implementation -- there's no way `RMT::Config.scc` could be missing: 1) it's marked as required in schema validation; 2) it's declared in the default config.

Reply:
Actually I implemented this because the tests were breaking without it. But I agree, I'll try other ways to mock or extend this class specifically to remove this method here

Reply:
Removed, just kept the implementation of the method falling back to `super` because of `rubocop`.
https://github.com/rubocop-hq/ruby-style-guide#no-method-missing